pages/_app.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
This is a custom App component for x0. All pages will be wrapped with whatever is in here